src/zap.c

```diff
@@ -5320,6 +5320,13 @@ makewish()
         buf[0] = '\0'; /* for EDIT_GETLIN */
         goto retry;
     }
+    if (buf[0] == '\0') {
+        if (yn("Really forfeit this wish?") == 'y') {
+            Strcpy(buf, "nothing");
+        }
+        else
+            goto retry;
```

Comment:
What if you truly want to wish for a random object? Can you just type "random" "random item" "anything"

Reply:
There is currently no way to wish for a random object. Not that it would be hard to add a special case for the string "random", but I really don't expect anyone to actually want a random object when they use a wish.